python/dgl/dataloading/shadow.py

```diff
@@ -0,0 +1,82 @@
+"""ShaDow-GNN subgraph samplers."""
+from ..utils import prepare_tensor_or_dict
+from ..base import NID
+from ..sampling import sample_neighbors
+from .neighbor import NeighborSamplingMixin
+from .dataloader import exclude_edges, Sampler
+
+class ShaDowKHopSampler(NeighborSamplingMixin, Sampler):
+    """K-hop subgraph sampler used by
+    `ShaDow-GNN <https://openreview.net/forum?id=GIeGTl8EYx>`__.
```

Comment:
It might be better to give the latest ArXiv link: https://arxiv.org/abs/2012.01380